test/browser_tests/page_objects/page_newsroom-press-resources.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var _assign = require( 'lodash' ).assign;
+var stayInformedSection = require( '../shared_objects/stayInformedSection' );
+var rssSection = require( '../shared_objects/rssSection' );
+
+function PressResources() {
+
+  _assign( this, stayInformedSection, rssSection );
+
+  this.get = function() {
+    browser.get( '/newsroom/press-resources/' );
+  };
+
+  this.pageTitle = browser.getTitle;
+
+  this.mainTitle = element( by.css( '[data-qa-hook="main-title"]' ) );
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/071c9fc3f5c87f92f74c956a1b34fa0c0d665156